pipelines/broad/dna_seq/germline/joint_genotyping/exome/JointGenotyping.inputs.json

```diff
@@ -23,7 +23,7 @@
   "JointGenotyping.mills_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi",
   "JointGenotyping.axiomPoly_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/Axiom_Exome_Plus.genotypes.all_populations.poly.hg38.vcf.gz.tbi",
 
-  "JointGenotyping.unpadded_intervals_file": "gs://gcp-public-data--broad-references/hg38/v0/exome_calling_regions.v1.interval_list",
+  "JointGenotyping.unpadded_intervals_file": "gs://broad-dsde-methods-gauthier/exome_joint_calling_regions.v1.1.interval_list",
```

Comment:
These should be copied into Google Public bucket if they are ready.  @kbergin

Reply:
@ldgauthier let me know when they are ready and I can at least get them into the broad public references bucket and then we can ask Google to update when it's all set.

Reply:
It covers like 99.9% of TWIST.  There are ~15 twist targets that aren't covered corresponding to the FAM231B
and CCDC32 genes (which are in RefSeq, but not gencode), but I didn't want to add anything new since we're not rerunning HaplotypeCaller.